dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/importers/ui/form/FormConstants.java

```diff
@@ -0,0 +1,11 @@
+package edu.harvard.iq.dataverse.importers.ui.form;
+
+public class FormConstants {
+    public static final ProcessingType[] SINGLE_OPTIONS
```

Comment:
Why not List?

Reply:
I thought that arrays will be more straightforward for this cases.